src/components/external/DisputeManagement/components/DisputesData/DisputeDataProperties.tsx

```diff
@@ -194,7 +202,7 @@ const DisputeDataProperties = ({ dispute, dataCustomization }: DisputeDataProper
                 }}
             />
         );
-    }, [dispute, dateFormat, dataCustomization?.details?.fields, extraFields]);
+    }, [dispute.payment, dispute.dispute, dispute.defense, i18n, dateFormat, extraFields, dataCustomization?.details?.fields]);
```

Comment:
I would recommend simplifying the dependencies here to this:
```suggestion
    }, [dispute, i18n, dateFormat, extraFields, dataCustomization?.details?.fields]);
```

Reason that the properties of the `dispute` object itself are not expected to change. And should that happen, it is rather an indication that something is not right.

Reply:
✅